constraints/miden-vm/hash.air

```diff
@@ -65,23 +77,9 @@ fn get_f_mua(s: vector[3]) -> scalar:
     return cycle_row_7 & s[0] & s[1] & s[2]
 
 
-# We can define two flags:
-# 1. Flag f_hout = cycle_row_7 & binary_not(s[0]) & binary_not(s[1]) & binary_not(s[2]), 
-# which is set to 1 when selector flags are (0,0,0) on rows which are 1 less than a multiple 
-# of 8. This is flag of the instruction that returns the result of the currently running 
-# computation.
-# 2. Flag f_sout = cycle_row_7 & binary_not(s[0]) & binary_not(s[1]) & s[2], which is set to 1 
-# when selector flags are (0,0,1) on rows which are 1 less than a multiple of 8. This is flag 
-# of the instruction that returns the whole hasher state.
-#
 # Flag f_out is set to 1 when either f_hout = 1 or f_sout = 1 in the current row.
 fn get_f_out(s: vector[3]) -> scalar:
-    return cycle_row_7 & binary_not(s[0]) & binary_not(s[1])
-
-
-# Flag f_out_next is set to 1 when either f_hout = 1 or f_sout = 1 in the next row.
-fn get_f_out_next(s: vector[3]) -> scalar:
-    return cycle_row_6 & binary_not(s[0]') & binary_not(s[1]')
+    return cycle_row_7 & !s[0] & !s[1]
```

Comment:
We can't actually make this change because these flags use different periodic columns